plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/CredentialsOptionsConfigurer.java

```diff
@@ -46,6 +46,7 @@ public BigQueryOptions.Builder configure(BigQueryOptions.Builder builder, Connec
         String billingProjectId = calculateBillingProjectId(parentProjectId, credentials);
         credentials.ifPresent(builder::setCredentials);
         builder.setProjectId(billingProjectId);
+        builder.setQuotaProjectId(billingProjectId);
```

Comment:
Might be a good idea to add this behind a flag, wdyt?

Reply:
@ebyhr can you take another look? also do you think it's a somoke test for this is required?